packages/react/script/components-json/build.ts

```diff
@@ -59,7 +60,12 @@ const components = docsFiles.map(docsFilepath => {
   const componentName = isComponentV2 ? `${docsName}v2` : docsName
 
   // Get the default story id
-  const defaultStoryId = `${storyPrefix[docs.status]}components-${componentName}--default`
+  let defaultStoryId = `${storyPrefix[docs.status]}components-${componentName}--default`
```

Comment:
I don't like this but couldn't think of another way to accommodate Dialog customisation. Let me know if you have any recommendation 🙌🏻